modules/admin/src/main/java/org/motechproject/admin/bundles/BundleDirectoryManager.java

```diff
@@ -22,6 +25,9 @@
 @Component
 public class BundleDirectoryManager {
 
+    @Autowired
+    private ConfigurationService configurationService;
+
     @Value("${user.home}/.motech/bundles")
     private String bundleDir;
```

Comment:
It would make more sense to init this in postConstruct instead of during each read.